spec/jobs/resolution_proofing_job_spec.rb

```diff
@@ -323,6 +326,30 @@
       end
     end
 
+    context 'with an invalid threatmetrix review_status value' do
+      it 'stores an exception result' do
+        threatmetrix_response = JSON.parse(LexisNexisFixtures.ddp_success_response_json).merge(
+          review_status: 'unexpected_review_status_that_causes_problems',
```

Comment:
Would it work to create a constant `BAD_REVIEW_STATUS = 'unexpected_review_status_that_causes_problems'` and then use it wherever it comes up in the tests?

Reply:
Can you help me understand the value a constant would provide? The valu is always used shortly after it is set. Subbing in a constant doesn't seem like it helps readability. As a constant the name may need to be even longer than this in order to disambiguate. Especially if it is going to be defined outside of the tests where it is used.

Reply:
Check out https://github.com/18F/identity-idp/pull/8149/commits/ae9369eb037cee76afa3319611e1c144d3c928bf and see if that helps to alleviate some of the concern.

Reply:
Yes, that works. My thought was to make it easier to see that the value being set and the value being compared against were the same.